gtdynamics/universal_robot/sdf.cpp

```diff
@@ -75,19 +75,18 @@ JointParams ParametersFromSdfJoint(const sdf::Joint &sdf_joint) {
 Pose3 GetJointFrame(const sdf::Joint &sdf_joint,
                     const LinkSharedPtr &parent_link,
                     const LinkSharedPtr &child_link) {
-  auto frame = sdf_joint.PoseFrame();
-  if (frame == "" || frame == child_link->name()) {
-    if (sdf_joint.Pose() == ignition::math::Pose3d())
+  auto frame = sdf_joint.PoseRelativeTo();
```

Comment:
Remove `auto` to show type, and change to `frame_name`

Reply:
Can we document this function? I don't understand what it is supposed to do without reading the code, and it is not in the .h file (nor should it be!).

Reply:
This is documented in `sdf_internal.h`.